cpp/src/replace/nulls.cu

```diff
@@ -352,6 +356,75 @@ std::unique_ptr<cudf::column> replace_nulls_scalar_kernel_forwarder::operator()<
   return cudf::dictionary::detail::replace_nulls(dict_input, replacement, stream, mr);
 }
 
+struct replace_nulls_replace_policy_functor {
```

Comment:
This name is awfully verbose.

Reply:
`replace_policy_functor`?

Reply:
Looking at other class/test names, there is certainly a better naming for them as well.